app/api/usergroups/userGroupsModel.ts

```diff
@@ -0,0 +1,15 @@
+import mongoose from 'mongoose';
+import { instanceModel } from 'api/odm';
+import { UserGroupSchema } from 'shared/types/userGroupType';
+
+const propsWithDBSpecifics = {
```

Comment:
I think it was a recommendation in the tech call to avoid duplicity of the schema definition, with this we only have the schema used for the AJV validation, @daneryl maybe did I understand wrong?

Reply:
@mfacar yes that is correct, thats the idea.